pkg/kubernetes/watcher.go

```diff
@@ -107,15 +107,15 @@ func NewObjListWatcher(bpfSupportedMetrics bpf.SupportedMetrics) *ObjListWatcher
 		return w
 	}
 	optionsModifier := func(options *metav1.ListOptions) {
-		options.FieldSelector = fmt.Sprintf("spec.nodeName=%s", stats.NodeName) // to filter events per node
+		options.FieldSelector = fields.Set{"spec.nodeName": stats.NodeName}.AsSelector().String() // to filter events per node
 	}
 	objListWatcher := cache.NewFilteredListWatchFromClient(
 		w.k8sCli.CoreV1().RESTClient(),
 		w.ResourceKind,
 		metav1.NamespaceAll,
 		optionsModifier,
 	)
-	w.informer = cache.NewSharedIndexInformer(objListWatcher, &corev1.Pod{}, 0, cache.Indexers{})
+	w.informer = cache.NewSharedIndexInformer(objListWatcher, &corev1.Pod{}, time.Minute*5, cache.Indexers{})
```

Comment:
This forces a full resync every 5 mins? Why?

Reply:
I thought it might be good to resync periodically - but I can set this back to 0